content/en/docs/concepts/windows/intro.md

```diff
@@ -352,6 +352,37 @@ Windows Server SAC release
 
 The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
 
+## Worker Node Hardware Recommendations {#windows-node-hardware-recommendations}
+
+The minimum hardware requirements for a Windows worker node depend on several
+factors, including the Windows operating system version and specific workload
+requirements. Mentioned below are the minimum system requirements, keeping in
+mind that they can vary based on the components installed on the node and the
+workloads required to run on it.
+
+- 1.4 GHz 64-bit processor, supporting virtualization
+- 8GB or more of RAM
+- NIC card with at least 1 Gbit/s bandwidth
+- At least 20GB of free disk space
+
+Refer to the specific Windows OS documentation for the most
+up-to-date information on minimum hardware requirements. For Windows Server
+installations, refer to
+[Hardware requirements for Windows Server](https://learn.microsoft.com/en-us/windows-server/get-started/hardware-requirements)
+Microsoft documentation for the bare minimum hardware requirements. Similarly,
+for Windows 10, refer
+to [Windows 10 system requirements](https://support.microsoft.com/en-us/windows/windows-10-system-requirements-6d4e9a79-66bf-7950-467c-795cf0386715).
```

Comment:
Either way we should leave out Windows 10 and re-target to Windows Server

Reply:
@aravindhp 
> I don't see similar documentation for Linux workloads

Would these count for that?
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin

Reply:
Good find @mloskot! So I take back my statement. We should add something similar for Windows 😄

Reply:
Sheer luck, @aravindhp, I have to admit that I was surprised to see it as part of the kubeadm docs :)

Reply:
+1 on removing Windows 10 (which is not supported for kubernetes) and focus on windows server

Reply:
Got it thanks all. I wil remove Windows 10 references. I actually think this is something that we could explicitly call out in the docs, I don't see it anywhere else listing the fact that Windows 10 is not supported but it is implicitly mentioned by saying worker nodes should be Windows Server.

Sidenote: Thanks @mloskot for the docs find regarding Linux!

Can I say something like the following. Will tweak and adjust later for better reading.
> Windows Sever is the only officially supported Windows operating system. While it may be possible to set up Kubernetes on other Windows operating systems such as Windows 10/11, they should only be used for development or testing purposes only. For production environments, only Windows Server is supported with Kubernetes.